peac/cpp/plane_fitter_pcd.cpp

```diff
@@ -229,7 +229,8 @@ void processOneFrame(pcl::PointCloud<pcl::PointXYZ>& cloud, const std::string& o
 
 int process() {
 	const double unitScaleFactor = global::iniGet<double>("unitScaleFactor", 1.0f);
-	const std::string outputDir = global::iniGet<std::string>("outputDir", ".");
+	std::string outputDirr= global::iniGet<std::string>("outputDir", ".");
+	const std::string outputDir=outputDirr.substr(0, outputDirr.size() - 1);
```

Comment:
2)On Linux platforms custom iniGet reads parameter with new line symbol at the end getting, I fixed it here